setup.cfg

```diff
@@ -20,7 +20,7 @@ classifiers =
 python_requires ~=3.8,<3.9
 install_requires =
     numpy==1.20.2
-    nltk==3.5.0
+    nltk==3.6.1
```

Comment:
why was this bumped?

Reply:
Dependabot keeps all of the dependencies up-to-date, but it only updates `requirements.txt` automatically. I forgot to update `setup.cfg` in the PR (https://github.com/wayfair-incubator/extra-model/pull/85), so updated it here

Reply:
👍